script/Ciel/Day 5/05_01_CIEL05_4.txt

```diff
@@ -862,25 +864,25 @@ He loved the flowers that bloomed in the snowfields,
 [sha:6a76b3e593c048ed63250d81c331de3ed2704d9e]{
 -- Page 724, Offset 12809.
 -- 男を慰める十人の<妃|つま>も手に入れた。
-He also acquired ten wives to console him.
+He even acquired ten wives to console him.
 }
 [sha:97a15cd067de6c0fa4e8b1c839f6c0695b654312]{
 -- Page 724, Offset 12810.
 -- 主君への忠誠も、決して溶けぬ氷壁と同義だった。
-His loyalty to his liege was like an ice wall that would never melt.
+The loyalty he held toward his liege was the same as the never-thawing ridges of ice.
 }
 [sha:076ea2d5b777e66336c59afb30541c6fcb7bc89b]{
 -- Page 725, Offset 12811.
 -- だが、魂に刻みついた痛みは癒えなかった。
-However, the pain carved into his soul could never be healed.
+Yet the pain carved into his soul would not heal.
 }
 [sha:f49f578ae8868927e5d581f64c0b4eb285bec253]{
 -- Page 725, Offset 12812.
 -- 寒さという名の乾きには抗えなかった。
 // 乾き short form dict entry is 'dryness', but it can also be parched / thirsty.
 // Rendering as thirst here, since I think that's consistent with the cold being a thirst for warmth (blood)
 // The phrasing makes it sound like he's thirsting for cold - would it be too much to replace 'the cold' with 'warmth'?
-He could not resist the thirst of 'being cold'.
+He could not oppose the parchedness that was his coldness.
```

Comment:
Or maybe reshuffle a bit 
> This was a cold---a thirst he could not endure.
> He could not endure this cold---this thirst.

Reply:
second reads super awkward but first is good

Reply:
I think all of these are overcomplications.

> He could not oppose the thirst of his cold.
Might be preferrable, in hindsight.
His cold thirsts after warmth, therefore, he seeks warmth.

Reply:
'his' cold reads a bit off to me since it kinda parses like he has the sniffles / possessive cold doesn't really make sense. If we want to attach it to him then I feel like you'd need more structure (the thirst of the cold _that {posessed,ruled,bound,other interpreted word} him_).

Reply:
Feels like a non-issue to me.
The context of this scene and the entire game should be enough to inform people that this is not that kind of cold.
Using the 'cold' as a noun like this will always be reminiscent of 'cold' in the sense of a viral infection, but it's not like that's any different from your suggestions ("This was a cold---a thirst he could not endure.") in that sense.

Reply:
I agree it's a minor point, but what stands out as different to me with that suggestions is that as _a_ cold I parse it as something that works upon him, rather than something he possesses

Reply:
> He could not oppose the thirst of the cold.

sounds fine to me